src/api.ts

```diff
@@ -20,7 +22,6 @@ export default function fetchWeatherForecast(
         let jsonValue = jsonResponse;
         if (
```

Comment:
I think you can use typescript to ensure that the jsonResponse should be a string, this will allow you to remove this if statement

Reply:
This is because the API sucks